src/main/flight/dyn_notch_filter.c

```diff
@@ -389,8 +391,9 @@ static FAST_CODE_NOINLINE void dynNotchProcess(void)
         {
             for (int p = 0; p < dynNotch.count; p++) {
                 // Only update notch filter coefficients if the corresponding peak got its center frequency updated in the previous step
+                float dt_compensation = schedulerGetCycleTimeMultiplier();
```

Comment:
You can pull this out of the loop
```suggestion
                const float dtCompensation = schedulerGetCycleTimeMultiplier();
```

Reply:
Would that pull it out of the loop? I should really just move it two lines up and outside of the loop.